docs/application/vstools/Tizen/import.md

```diff
@@ -0,0 +1,25 @@
+# Importing Tizen Studio projects into Visual Studio
```

Comment:
1st Headings should follow Title Case.
```suggestion
# Importing Tizen Studio Projects Into Visual Studio
```

Reply:
Thanks @safir-srbd. Adding these changes to new patch